byterun/misc.c

```diff
@@ -63,21 +63,21 @@ void caml_gc_message (int level, char *msg, uintnat arg)
 CAMLexport void caml_fatal_error (char *msg)
 {
   fprintf (stderr, "%s", msg);
-  exit(2);
+  abort();
```

Comment:
This is a fairly large semantic change to `caml_fatal_error`, as `abort` will send `SIGABRT` and not cause `atexit` handlers to to be invoked.